src/qt/masternodelist.cpp

```diff
@@ -455,7 +455,7 @@ void MasternodeList::updateDIP3List()
             if (dmn->pdmnState->scriptOperatorPayout != CScript()) {
                 CTxDestination dest;
                 if (ExtractDestination(dmn->pdmnState->scriptOperatorPayout, dest)) {
-                    operatorRewardStr += tr(" to %1").arg(QString::fromStdString(CBitcoinAddress(payeeDest).ToString()));
+                    operatorRewardStr += tr(" to %1").arg(QString::fromStdString(CBitcoinAddress(dest).ToString()));
```

Comment:
Oh, wow, how did this one slip through? 🙈 Let's maybe rename `dest` to `operatorDest` to be 100% explicit/consistent?

Reply:
done